pages/linux/truncate.md

```diff
@@ -0,0 +1,15 @@
+# truncate
+
+> Shrink or extend the size of a file to the specified size.
+
+- Set the size of an existing file, or create a new file with the specified size:
```

Comment:
We should mention the size here. `Set a size of 10 GB to an exsting file, or .. `